packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -1,6 +1,121 @@
 'use strict'
 
-const unified = require('./unified')
-const legacy = require('./legacy')
+const Plugin = require('../../dd-trace/src/plugins/plugin')
+const { storage } = require('../../datadog-core')
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+class MongodbCorePlugin extends Plugin {
+  static get name () {
+    return 'mongodb-core'
+  }
 
-module.exports = [].concat(unified, legacy)
+  addSubs (func, start, asyncEnd = defaultAsyncEnd) {
```

Comment:
There's only ever one asyncEnd added here, so no need to have it as an argument.